src/screens/Homepage.tsx

```diff
@@ -1,12 +1,102 @@
-        </Box>
+        <>
+
+            <Box p={'4'} bg={'#F5F5F5'}>
+                <Center >
+                    <Flex alignItems={'center'}>
+                        <Badge mx={'10px'} bg={'blue.600'} color={'#FFF'}>OFFER</Badge>
+                        <Text><b>Need It Today?</b> Free In Store & Pickup / <b>$9.99</b> Same Day Delivery</Text>
+                    </Flex>
+                </Center>
+            </Box>
+
+            <Box>
+                <Slider />
+            </Box>
+
+            <Box px={'100px'} pb={'50px'} borderBottom={'1px solid #ccc'}>
+                <Flex>
+                    {/* how to reduce it and thanks */}
+                    <Box width={'100%'} py={'60px'} _hover={{ bg: '#f0f0f0', transition: 'all .5s ease-in-out' }}>
+                        <Center flexDirection={'column'}>
+                            <Box py={'1.5'}><FiTruck color={'#f7d929'} fontSize={'50px'} /></Box>
+                            <Text py={'1.5'} fontSize={'16px'} fontWeight={'700'}>FAST FREE DELIVERY</Text>
+                            <Text color={'gray'} py={'1.5'} textAlign={'center'} maxW={'250px'}>{pargraph}</Text>
+                            <Link textDecoration={'underline'} fontWeight={'600'} py={'1.5'}>Read More</Link>
+                        </Center>
+                    </Box>
+
+                    <Box width={'100%'} py={'60px'} _hover={{ bg: '#f0f0f0', transition: 'all .5s ease-in-out' }}>
+                        <Center flexDirection={'column'}>
+                            <Box py={'1.5'}><BiMessageDetail color={'#f7d929'} fontSize={'50px'} /></Box>
+                            <Text py={'1.5'} fontSize={'16px'} fontWeight={'700'}>ONLINE 24/7 SUPPORTS</Text>
+                            <Text color={'gray'} py={'1.5'} textAlign={'center'} maxW={'250px'}>{pargraph}</Text>
+                            <Link textDecoration={'underline'} fontWeight={'600'} py={'1.5'}>Read More</Link>
+                        </Center>
+                    </Box>
+
+                    <Box width={'100%'} py={'60px'} _hover={{ bg: '#f0f0f0', transition: 'all .5s ease-in-out' }}>
+                        <Center flexDirection={'column'}>
+                            <Box py={'1.5'}><RiSecurePaymentLine color={'#f7d929'} fontSize={'50px'} /></Box>
+                            <Text py={'1.5'} fontSize={'16px'} fontWeight={'700'}>100% SECURE PAYMENT</Text>
+                            <Text color={'gray'} py={'1.5'} textAlign={'center'} maxW={'250px'}>{pargraph}</Text>
+                            <Link textDecoration={'underline'} fontWeight={'600'} py={'1.5'}>Read More</Link>
+                        </Center>
+                    </Box>
+
+                </Flex>
+            </Box>
+
+
+            <Box>
+                {dataList.map(function (data) {
+                    const { id, product, summary, longLine } = data;
+                    return (
+                        <Card
+                            key={id}
```

Comment:
Nicccce, I am happy you used Key prop. 

Read more about it here: 
https://reactjs.org/docs/lists-and-keys.html